server/protocols/protocols-smtp/src/main/java/org/apache/james/smtpserver/SendMailHandler.java

```diff
@@ -70,6 +70,8 @@ public HookResult onMessage(SMTPSession session, Mail mail) {
         LOGGER.debug("sending mail");
 
         try {
+            Message message = mail.getMessage();
+            message.setSubject(mail.getMessage().getSubject());
```

Comment:
Given the change to load headers using UTF-8 is this necessary?

Reply:
I don't thinks it IS necessary in my experience, because it looks like the codes do nothing but just re-set the subject of the message...........
However, If I remove this two lines, the error code of subject issue will not be solved. I don't know why, do you have any other suggestions?